src/components/pages/roster-page/roster-page.tsx

```diff
@@ -142,16 +183,134 @@ export const RosterPage = () => {
     setRowsPerPage(newRowsPerPage);
   };
 
-  const handleAlertClose = () => {
-    setAlert({ open: false, message: '', title: '' });
+  const editButtonClicked = async (rosterEntry: ApiRosterEntry) => {
+    // remove any properties that came back as null before sending it to the dialog
+    Object.keys(rosterEntry).forEach((key: string) => (rosterEntry[key] == null) && delete rosterEntry[key]);
```

Comment:
I ran into this thing where when the server responds with the roster entries, if they don't have a value for a column it ends up as null. Then when we go to edit and save an entry, an error gets thrown because a value was null.

I'm handling that here right now by just removing the entries that are null (they'll get added back if they're edited) which works fine. With that said, I'm not sure if this is the best place or the right way to handle that.

Reply:
Awesome, just tested it and it looks like this works now. I'll remove this check.